app/controllers/transport/border/active/AddAnotherBorderTransportController.scala

```diff
@@ -52,23 +57,32 @@ class AddAnotherBorderTransportController @Inject() (
       viewModel.count match {
         case 0 =>
           Redirect(
-            controllers.transport.border.routes.BorderModeOfTransportController.onPageLoad(departureId, mode)
-          ) //TODO: Change to /add-identification when built
+            routes.IdentificationController.onPageLoad(departureId, mode, Index(0))
```

Comment:
The TODO seems to suggest this should be going to the /add-identification page, but this looks like the /identification page? Was the TODO incorrect? Is there a page missing?

Reply:
@jackheslop96 spoke to Sayak this  page can load up regardless of how many there are and  show zero if the last one was removed. Code updated, Messaged Gavin and we may have a separate ticket for dynamic content if there are zero

Reply:
Following discussions. Removed the Remove link for when there is only one BorderMeansOfTransport if user has more than one and removes one then confirm remove goes back to the add another page.